pkg/client/userd/daemon/grpc.go

```diff
@@ -155,20 +155,23 @@ func (s *Service) Status(ctx context.Context, ex *empty.Empty) (result *rpc.Conn
 			})
 		} else {
 			result = s.session.Status(s.sessionContext)
-			stts, err := s.session.ManagerClient().Status(c, &empty.Empty{})
+			var stts *manager.StatusInfo
+			stts, err = s.session.ManagerClient().Status(c, &empty.Empty{})
 			if err != nil {
 				stat, ok := status.FromError(err)
 				if !ok {
 					return
 				}
 				if stat.Code() == codes.Unimplemented {
```

Comment:
As I pointed out earlier:
```suggestion
				if status.Code(err) == codes.Unimplemented {
```

Reply:
missed that